terraform/terraform.tfvars.example

```diff
@@ -0,0 +1,12 @@
+environment                 = "development"
+project_name                = "myproject"
+azure_location              = "uksouth"
+enable_container_registry   = true
+image_name                  = "myimage"
+enable_mssql_database       = true
+mssql_server_admin_password = "S3crEt"
+mssql_database_name         = "mydatabase"
+container_command           = ["/bin/bash", "-c", "echo hello && sleep 86400"]
+container_environment_variables = {
+  "RAILS_ENV"                       = "production"
```

Comment:
We're not using Rails. In asp.net traditionally you would use -ASPNETCORE_ENVIRONMENT

Reply:
there may be other too you want to wham into env vars

Reply:
Ah yup ... Copypasta ...